src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertTooltip.tsx

```diff
@@ -45,13 +30,13 @@ const ArtworkAuctionCreateAlertTooltip: React.FC<ArtworkAuctionCreateAlertToolti
             {pills.map(pill => {
               return (
                 <Pill
-                  key={pill.id}
+                  key={`pill-${pill.field}-${pill.value}`}
                   variant="filter"
                   disabled
                   width="auto"
                   mr={0.5}
```

Comment:
This should also be `mr={1}`, just realised.

Reply:
Just use `gap` on the parent